circle.yml

```diff
@@ -5,8 +5,27 @@ test:
   pre:
     - NODE_ENV=test make client/config/index.js:
         parallel: true
-    - case $CIRCLE_NODE_INDEX in 0) NODE_ENV=test make build-server ;; esac:
-        parallel: true
+    - ? |
+          case $CIRCLE_NODE_INDEX in
+            0) NODE_ENV=test make build-server;
+               ;;
+
+            1) I18N_DIR=$CIRCLE_ARTIFACTS/translate;
+               mkdir -p $I18N_DIR;
```

Comment:
I think they can be taken off but I ended up leaving them just to be safe. I ran into some trouble early on with yaml + bash syntax complexity so I've erred toward explicit termination.